modules/programs/floorp.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, ... }:
+
+with lib;
+
+let
+
+  modulePath = [ "programs" "floorp" ];
+
+  mkFirefoxModule = import ./firefox/mkFirefoxModule.nix;
+
+in {
+  meta.maintainers = [ hm.maintainers.bricked ];
+
+  imports = [
+    (mkFirefoxModule {
+      inherit modulePath;
+      name = "Floorp";
+      wrappedPackageName = "floorp";
+      unwrappedPackageName = "floorp-unwrapped";
+      visible = true;
+
+      platforms.linux = {
+        configPath = ".floorp";
+        vendorPath = ".floorp";
```

Comment:
I don't think the vendorPath can overlap with the configPath. Not sure if Floorp supports native messaging hosts anyways, so this might be redundant.